roles/openshift_node/meta/main.yml

```diff
@@ -15,3 +15,8 @@ dependencies:
 - role: openshift_docker
 - role: openshift_cloud_provider
 - role: openshift_common
+- role: openshift_node_dnsmasq
+  dns_domain: "{{ openshift.common.dns_domain }}"
+  kube_svc_ip: "{{ openshift.common.kube_svc_ip }}"
+  when: openshift_node_dnsmasq | default(True if openshift.common.version_gte_3_2_or_1_2 | bool else False) | bool
```

Comment:
Since I need to know if we're using dnsmasq in master role too should I just move this to openshift.common.use_dnsmasq ?